docs/HACKING.md

```diff
@@ -55,13 +56,15 @@ Additional flags that can be passed to `hack/run-ci-e2e-test.sh` are
 - `-b` gives an alternative path to the WMCO binary. This option overridden in OpenShift CI.
        When building the operator locally, the WMCO binary is created as part of the operator image build process and
        can be found at `build/_output/bin/windows-machine-config-operator`, this is the default value of this option.
+- `-t` specify the test to run. All tests are run if the option is not used. The allowed options are:
+  - `all` all the tests are run
+  - `basic` creation, network and deletion tests are run
+  - `upgrade` creation, upgrade, reconfiguration and deletion tests are run
 
-       
 Example command to spin up 2 Windows nodes and retain them after test run:
 ```shell script
 hack/run-ci-e2e-test.sh -s -n 2      
 ```
-
```

Comment:
can add this line back

Reply:
I removed it on purpose as I was editing that area. Is there a reason you want that extra line?

Reply:
doesn't seem like we need it from the rich diff, lgtm